physcraper/__init__.py

```diff
@@ -1683,8 +1617,60 @@ def __init__(self, data_obj, ids_obj):
         self.backbone = False
         self.OToL_unmapped_tips()  # added to do stuff with un-mapped tips from OToL
         self.ids.ingroup_mrca = data_obj.ott_mrca  # added for mrca ingroup list
-        self.gb_not_added = []  # list of blast seqs not added
+        #self.gb_not_added = set()  # list of blast seqs not added
         self.del_superseq = {}  # items that were deleted bc they are superseqs, needed for assert statement
+        self.mrca_ott_list = ingroup_mrca
+        assert type(self.mrca_ott_list) in [int, list] or self.mrca_ott_list is None
+        self.mrca_ncbi_list = set()  # corresponding ids for mrca_ott list
+        if self.mrca_ott_list is not None:
+            self.get_ncbi_mrca()
+        # self.mrca_ncbi_of_all = ids_obj.ott_to_ncbi[data_obj.ott_mrca]
+        if len(self.mrca_ncbi_list) == 0:
```

Comment:
This should also happen if self.mrca_ott_list == None, right? So that if no mrca is passed in at all, it will use the one from the data_object.  Maybe only if if self.mrca_ott_list == None, because if someone passed something in and the length is 0, that is probably an error.

Reply:
So, if we have an ingroup mrca specified by the user, then self.mrca_ott_list is not None and we will translate the ids to ncbi ids. If there is nothing specified, the self.mrca_ncbi_list was not modified and the length of the set() is zero and we will add the translated value of data_obj.ott_mrca.
So, I think what you're suggesting is already in the code.

Reply:
Ah right! I see now.